test/utils/client.py

```diff
@@ -80,11 +80,11 @@ class Hwm:
     """Class representing app high water mark."""
 
     highest_level: int
-    highest_round: Optional[int]
+    highest_round: int
 
     def __init__(self,
                  highest_level: int,
-                 highest_round: Optional[int] = None,
+                 highest_round: int = 0,
```

Comment:
Let's not have a default value for the round, because it's important to be explicit when we create a `HWM`.

Reply:
The round value is appended to response from the app for command get_hwm, if the protocol is upgraded to tenderbake. To remove default value 0 we will have to change behaviour of get_hwm in the app. What do you think? Should i not append round value to HWM if round value is zero. I feel keeping the round value seems more appropriate, we can indicate that HWM referes to block and round.